diff --git a/schemas/__init__.py b/schemas/__init__.py
index 7794dc0caf2e48bba941b0758c5d5b704a6017fc..9e8665420a6584ecd68bb00b6ef1cad4a57daca4 100644
--- a/schemas/__init__.py
+++ b/schemas/__init__.py
@@ -85,25 +85,26 @@ class QuestionCreate(QuestionBase):
 class Question(QuestionBase):
     id: int
 
     class Config:
         orm_mode = True
 
 
 class OptionBase(BaseModel):
     option_text: str
 
 
 class OptionCreate(OptionBase):
     pass
 
 
 class Option(OptionBase):
     id: int
     question_id: int
 
     class Config:
         orm_mode = True
 
 
 from .answer import Answer, AnswerCreate
 from .annotation import Annotation, AnnotationCreate, AnnotationUpdate
+from .expert_type import ExpertType, ExpertTypeBase, ExpertTypeCreate
diff --git a/schemas/expert_type.py b/schemas/expert_type.py
new file mode 100644
index 0000000000000000000000000000000000000000..bebff019b54b533b5e0af51cbf386fa9bf8d3a43
--- /dev/null
+++ b/schemas/expert_type.py
@@ -0,0 +1,16 @@
+from pydantic import BaseModel
+
+
+class ExpertTypeBase(BaseModel):
+    name: str
+
+
+class ExpertTypeCreate(ExpertTypeBase):
+    pass
+
+
+class ExpertType(ExpertTypeBase):
+    id: int
+
+    class Config:
+        orm_mode = True
diff --git a/schemas/user.py b/schemas/user.py
index ff6f416c13afc67ccafe2ed1e89898c55097f8cf..bc04641a5f4d4e5f19f6496c1e17db8a805959b3 100644
--- a/schemas/user.py
+++ b/schemas/user.py
@@ -1,31 +1,34 @@
 from pydantic import BaseModel
 
+from .expert_type import ExpertType
+
 
 class UserBase(BaseModel):
     username: str
     role: str = "Esperto"
 
 
 class UserCreate(UserBase):
     password: str
 
 
 class UserLogin(BaseModel):
     username: str
     password: str
 
 
 class UserResponse(UserBase):
     id: int
+    expert_types: list[ExpertType] = []
 
     class Config:
         orm_mode = True
 
 
 class Token(BaseModel):
     access_token: str
     token_type: str
 
 
 class TokenData(BaseModel):
     username: str | None = None
